SyliusShopBundle/views/Product/Index/_pagination.html.twig

```diff
@@ -1,5 +1,7 @@
 {% set paginationLimits = (resources.definition.limits) %}
 
+<div class="d-flex justify-content-end my-3">
```

Comment:
Tag unclosed in this file ... 
You 'll get an error if you disable the pagination block into config

Should be moved to _main.html.twig ?